src/routes/api/installation/api-installation-delete.ts

```diff
@@ -6,6 +6,11 @@ export const ApiInstallationDelete = async (req: Request, res: Response): Promis
 	const githubInstallationId = req.params.installationId;
 	const jiraHost = req.params.jiraHost;
 
+	//TODO: ARC-1619 Maybe need to fix this and put it into the path
+	//Not doing it now as it might break pollinator if it use this api
+	const { gitHubAppIdStr } = req.query;
```

Comment:
is the value set in the query actually `gitHubAppIdStr`?  We shouldn't be using query for entity information as it breaks the REST conventions - it should be part of the url params.

Reply:
also, why is this needed in the API?  It's kinda outside the scope of this PR

Reply:
Actually I've remove/revert the api changes here. Created new ticket to tackle that ARC-1622